src/render/tutorials/caching-http-requests/2_0.html.md

```diff
@@ -0,0 +1,314 @@
+    handlers = ${config.server.handlers.common.request} [
+      {
+        name = singleFragmentSupplier
+        config = {
+          type = json
+          configuration {
+            data-knotx-task = api-task
+          }
+        }
+      },
+      {
+        name = fragmentsHandler
+        config = {include required(classpath("routes/handlers/fragments-handler.conf"))}
+      },
+
+      {
+        name = fragmentsAssembler
+      }
+    ] ${config.server.handlers.common.response}
+  }
+  {
+    operationId = healthcheck-operation
+    handlers = [
+      {
+        name = healthcheck
+      }
+    ]
+  }
+]
+```
+Regarding mentioned operation, we can see, that we're using three different handlers to perform it.
+In our case, first handler to be called is `singleFragmentSupplier`. It's used to convert incoming http request
+into `knotx-fragment` and assign `data-knotx-task` to it. Next to be called is `fragmentsHandler`, here we have to attach
+additional configs with tasks specification. The last one is `fragmentsAssembler`
+Purpose of using it, is to combine all fragments into final response. And this is all about operations.
+
+## OpenAPI
+Finally we can create `./knotx/conf/openapi.yaml`, it contains project-specific API definitions using the [OpenAPI 3.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md) syntax. 
+Let's fill it out with:
+```yaml
+openapi: "3.0.0"
+info:
+  version: 1.0.0
+  title: API gateway caching example
+  description: API gateway caching example
+
+servers:
+  - url: https://{domain}:{port}
+    description: The local API server
+    variables:
+      domain:
+        default: localhost
+        description: api domain
+      port:
+        enum:
+          - '8092'
+        default: '8092'
+
+paths:
+  /healthcheck:
+    get:
+      operationId: healthcheck-operation
+      responses:
+        default:
+          description: example vert.x healthcheck
+
+  /product/id:
+    get:
+      operationId: product-api-caching-proxy-operation
+      responses:
+        default:
+          description: Target API (Product API) caching proxy.
+```
+This is whole structure of this file required in our case. In more detail it was explained in [this tutorial](http://knotx.io/tutorials/getting-started-with-knotx-stack/2_0/) 
+The most important for us here is `/product/id` endpoint definition. As you can see we are connecting this endpoint with
+`product-api-caching-proxy-operation` which we've defined before. Moreover here we've provided some other specification 
+but it was described in other tutorials, so if you don't feel familiar with it, please check them out.
+
+We have all Knot.x specific files created, now we are gonna focus on project setup and deployment.  
+
+## Project configuration
+The first thing we need to do is to specify configuration files. Let's start in the project level directory and simply create `api-cache.yml`
```

Comment:
A few words on what file we're creating here (Docker swarm config) would be nice.

Reply:
fixed